I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/scripts/initialise/classes.rpy", line 1, in script
    init python:
  File "game/scripts/initialise/classes.rpy", line 1, in script
    init python:
  File "game/scripts/initialise/classes.rpy", line 24, in <module>
    Rooms.append(place("Cozinha",0,True))
  File "game/scripts/initialise/classes.rpy", line 4, in __init__
    self.name = name
AttributeError: can't set attribute

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/bootstrap.py", line 331, in bootstrap
    renpy.main.main()
  File "renpy/main.py", line 560, in main
    renpy.game.context().run(node)
  File "game/scripts/initialise/classes.rpy", line 1, in script
    init python:
  File "game/scripts/initialise/classes.rpy", line 1, in script
    init python:
  File "renpy/ast.py", line 928, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "renpy/python.py", line 2245, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/scripts/initialise/classes.rpy", line 24, in <module>
    Rooms.append(place("Cozinha",0,True))
  File "game/scripts/initialise/classes.rpy", line 4, in __init__
    self.name = name
  File "renpy/python.py", line 875, in do_mutation
    return method(self, *args, **kwargs)
  File "renpy/python.py", line 1210, in __setattr__
    object.__setattr__(self, attr, value)
AttributeError: can't set attribute

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
Projeto Teste 1.0
Sun Jun 26 12:11:36 2022
